cfgov/unprocessed/css/organisms/post-preview.less

```diff
@@ -102,20 +131,49 @@
                 vertical-align: bottom;
             }
         } );
+
+        &_left {
+            .u-webfont-medium();
+            display: inline-block;
+            float: none;
+            font-size: 1.125em;
+            grid-area: category;
+            line-height: 1.33333;
+
+            .m-meta-header_separator {
+                padding-left: 0.25em;
+                padding-right: 0.25em;
+            }
+        }
+
+        &_right {
+            display: inline-block;
+            float: none;
+            grid-area: date;
+            line-height: 1.5;
+
+            .a-date {
+                line-height: 1.7143;
```

Comment:
This value looks suspect too. Magic number. How's this calculated?

Reply:
It's 24/14 - `line-height` of 24px, `font-size` of 14px. Is there a better/more preferred way for us to express that?

Reply:
Changing this to `unit(24px / 14px)`